operator/src/network/ipfs_rpc.rs

```diff
@@ -14,7 +14,7 @@ pub trait IpfsRpcClient {
 /// Information about connected peers
 #[derive(Debug, Clone, serde::Deserialize)]
 pub struct Peer {
-    pub addr: String,
+    #[serde(rename = "Peer")]
```

Comment:
What's the deal here? I see `Peer` as a field in [the kubo docs](https://docs.ipfs.tech/reference/kubo/rpc/#api-v0-swarm-peers) but I'm confused about why this was needed (or how this was fine before since it seems like it should have been `Id` and `Addr`).

Reply:
Yeah, I can see how that would be confusing haha.

Before the previous PR for filtering on Keramik peer IDs, we actually didn't care about _any_ of the fields in that struct:
```
        #[derive(serde::Deserialize)]
        struct Peer {
            //We currently do not care about any of the information of connected peers,
            // just the number of them. As a result this struct does not have any fields.
        }
        #[derive(serde::Deserialize)]
        struct Response {
            #[serde(rename = "Peers")]
            peers: Option<Vec<Peer>>,
        }
```
Then the last PR started caring about the fields:
```
/// Information about connected peers
#[derive(Debug, Clone, serde::Deserialize)]
pub struct Peer {
    pub addr: String,
    pub id: String,
}
```
However, there was a bug in my code. I changed the field name to `id`, when in fact, the API returns `peer` 🙄

With this latest change, we removed `addr` because we don't really care about it, and are deserializing the peer ID correctly. I think `id` is clearer than `peer`.

Reply:
Oh right, that makes sense. We used to just count elements in the list and now we actually check the id. Thanks for explaining!